### PR TITLE
[Merged by Bors] - chore: move `build.yml.in` to `../build.in.yml`

### PR DESCRIPTION
**Pull request by adomani**
The main motivation is to use the "correct" extension and get syntax highlighting and better editing experience for this file!

The reason to move it one dir up, is that otherwise it gets picked up by GitHub CI as a possible workflow.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Makes sense to me. Can you add a comment to `build.in.yml`, pointing to the other files in the `.workflows` directory? (Sorry, this is pre-existing. Feel free to ignore as scope creep.)

maintainer merge

**Comment by grunweg** [comment]
Let's try again
maintainer merge

File: .github/build.in.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# This is the master file for autogenerating `.github/workflows/{bors, build_fork  build }.yml`
```
Comment (grunweg) [inline .github/build.in.yml:1]:
```suggestion
# This is the master file for autogenerating `.github/workflows/{bors, build_fork, build }.yml`.

Comment (adomani) [reply .github/build.in.yml]: Thanks!

**Comment by bryangingechen** [comment]
Thanks!
bors r+

**Pull request closed without merging**